src/client/sandbox/code-instrumentation/index.js

```diff
@@ -30,6 +31,24 @@ export default class CodeInstrumentation extends SandboxBase {
         this.locationAccessorsInstrumentation.attach(window);
         this.elementPropertyAccessors = this.propertyAccessorsInstrumentation.attach(window);
 
-        window[PROCESS_SCRIPT_METH_NAME] = script => typeof script !== 'string' ? script : processScript(script);
+        window[INSTRUCTION.processScript] = (script, isApply) => {
+            if (isApply) {
+                if (script && script.length && typeof script[0] === 'string') {
+                    var args = [processScript(script[0])];
+
+                    // NOTE: shallow copy rest of the args. Don't use arr.slice(),
+                    // since it may cause memory leak with `arguments` object.
+                    // See: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/arguments
```

Comment:
NOTE: shallow-copy the remaining args. Don't use arr.slice(), since it may leak the `arguments` object.
See: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/arguments